### PR TITLE
[v14] fix(jest-preset): missing polyfill for plural rules

### PR DESCRIPTION
**Pull request by emmenko**
TL;DR; the polyfill for `PluralRules` was missing (for nodejs/test environment).

### CODE REVIEW

File: packages/jest-preset-mc-app/polyfills/intl-pluralrules.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/**
+ * NOTE: this is a copy of https://github.com/eemeli/intl-pluralrules
+ * as the package is shipped with ES6 code.
+ */
+const pluralRules = require('make-plural');
```
Comment (emmenko) [inline packages/jest-preset-mc-app/polyfills/intl-pluralrules.js]: Basically changing the imports to cjs

File: packages/jest-preset-mc-app/polyfills/intl-pluralrules.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  }
+
+  select(number) {
+    if (typeof number !== 'number') number = Number(number);
+    if (!isFinite(number)) return 'other';
+    const fmt = this._format(Math.abs(number));
+    return pluralRules[this._locale](fmt, this._type === 'ordinal');
+  }
+}
+module.exports = PluralRules;
```
Comment (emmenko) [inline packages/jest-preset-mc-app/polyfills/intl-pluralrules.js]: ...and the export

File: packages/jest-preset-mc-app/polyfills/intl.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Polyfill `Intl` for NodeJS, as `react-intl` (v3) relies on the `intl-locales-supported`
+// package, which checks if the locale is supported by the following constructors.
+// In the browser everything is fine, however in NodeJS environment we need to polyfill it.
+// https://github.com/formatjs/formatjs/tree/master/packages/intl-messageformat#modern-intl-dependency
+global.Intl = require('intl');
+global.Intl.PluralRules = require('./intl-pluralrules');
```
Comment (emmenko) [inline packages/jest-preset-mc-app/polyfills/intl.js]: FYI: we didn't catch this problem before because in appkit we don't use plural rules.

**Comment by emmenko** [comment]
I'll merge and release `14.0.0-alpha.1` so that I can continue with the migration.

**Comment by tdeekens** [comment]
What on earth is this? What does this do?

**Comment by emmenko** [comment]
A plural rules is when you define a message like this:

```
{count} {count, plural, one {This is one} other {Those are more than one}}
```

**Comment by emmenko** [comment]
The `PluralRules` polyfill is not included in the `intl` package.

https://github.com/formatjs/formatjs/tree/master/packages/intl-messageformat#modern-intl-dependency

**Comment by tdeekens** [comment]
Ah that stuff.

**Pull request merged**